src/shmessy/types/__init__.py

```diff
@@ -1,19 +1,27 @@
 import logging
-from datetime import datetime
 
-from numpy import ndarray
+from shmessy.types.base import BaseType
+from shmessy.types.boolean import BooleanType
+from shmessy.types.date import DateType
+from shmessy.types.datetime_ import DatetimeType
+from shmessy.types.email import EmailType
+from shmessy.types.float import FloatType
```

Comment:
As I mentioned in my other comment. I am reading the types dynamically. this piece of code broke this concept...